ksql-parser/src/test/java/io/confluent/ksql/parser/rewrite/StatementRewriterUnitTest.java

```diff
@@ -0,0 +1,624 @@
+package io.confluent.ksql.parser.rewrite;
```

Comment:
I am planning on replacing StatementRewriterTest with this test. Wanted to get a green run with the old tests first.